App.js

```diff
@@ -58,201 +64,216 @@ const originalSend = XMLHttpRequest.prototype.send;
-        Settings: { screen: Settings },
-    },
-    StackNavigatorOptions
+	{
+		Profile: { screen: Profile },
+		Settings: { screen: Settings },
+	},
+	StackNavigatorOptions
 );
 
 const ToolNavigator = createStackNavigator(
-    {
-        DiagnosticTool: { screen: DiagnosticTool },
-        DiagnosticToolResults: { screen: DiagnosticToolResults },
-        DiagnosisDetailView: { screen: DiagnosisDetailView },
-    },
-    StackNavigatorOptions
+	{
+		DiagnosticTool: { screen: DiagnosticTool },
+		DiagnosticToolResults: { screen: DiagnosticToolResults },
+		DiagnosisDetailView: { screen: DiagnosisDetailView },
+	},
+	StackNavigatorOptions
 );
 
 const PetsNavigator = createStackNavigator(
-    {
-        Pets: { screen: Pets },
-        PetDetailView: { screen: PetDetailView },
-        AddPets: { screen: AddPets },
-        TrainingScreen: { screen: TrainingScreen },
-        EditScreen: { screen: EditScreen },
-        ViewDocuments: { screen: ViewDocuments },
-        PetPrescription: { screen: PetPrescription },
-        PetDiet: { screen: PetDiet },
-    },
-    StackNavigatorOptions
+	{
+		Pets: { screen: Pets },
+		PetDetailView: { screen: PetDetailView },
+		AddPets: { screen: AddPets },
+		TrainingScreen: { screen: TrainingScreen },
+		EditScreen: { screen: EditScreen },
+		ViewDocuments: { screen: ViewDocuments },
+		PetPrescription: { screen: PetPrescription },
+		PetDiet: { screen: PetDiet },
+	},
+	StackNavigatorOptions
+);
+
+const ChatNavigator = createStackNavigator(
+	{
+		ChatTest: { screen: ChatTest },
+	},
+	StackNavigatorOptions
 );
 
 const PatientsNavigator = createStackNavigator(
-    {
-        Patients: {screen: Patients},
-        Pets: {screen: Pets},
-        PetDetailView: {screen : PetDetailView},
-        AddPets: { screen: AddPets },
-        TrainingScreen: { screen: TrainingScreen },
-        EditScreen: { screen: EditScreen },
-        ViewDocuments: { screen: ViewDocuments },
-        PetPrescription: { screen: PetPrescription },
-        PetDiet: { screen: PetDiet }
-    },
-    StackNavigatorOptions
+	{
+		Patients: { screen: Patients },
+		Pets: { screen: Pets },
+		PetDetailView: { screen: PetDetailView },
+		AddPets: { screen: AddPets },
+		TrainingScreen: { screen: TrainingScreen },
+		EditScreen: { screen: EditScreen },
+		ViewDocuments: { screen: ViewDocuments },
+		PetPrescription: { screen: PetPrescription },
+		PetDiet: { screen: PetDiet },
+	},
+	StackNavigatorOptions
 );
 
 const HomeNavigator = createBottomTabNavigator(
-    {
-        Pets: { screen: PetsNavigator },
-        DiagnosticTool: { screen: ToolNavigator },
-        Profile: { screen: ProfileNavigator },
-    },
-    {
-        animationEnabled: true,
-        tabBarComponent: HomeTab,
-        tabBarPosition: "bottom",
-        swipeEnabled: false,
-    }
+	{
+		Pets: { screen: PetsNavigator },
+		DiagnosticTool: { screen: ToolNavigator },
+		Profile: { screen: ProfileNavigator },
+		Chat: { screen: ChatNavigator },
```

Comment:
Somewhere along the line when merging with other files or branches, ChatTest got added back into this file, and is causing the app to crash instantly since there is no ChatTest.js file anywhere, all occurrences of Chat/ChatTest/ChatNavigator must be removed from this file again

Reply:
Yeah, I also noticed but was going to bring it up in the meeting just to double check.